dataplane/protocol/registry.go

```diff
@@ -62,6 +63,11 @@ func NewRegistry(psc pktiopb.PacketIO_CPUPacketStreamClient) (*Registry, error)
 	return pr, nil
 }
 
+// Context returns the context of psc.
```

Comment:
nit: instead of psc say the grpc client.

Reply:
done.